### ISSUE TITLE
A11y Collapse Element FE UI: Contrast between background and Link text is too low

### ISSUE THREAD

**Issue by TaylorYeeBC** [BellevueCollege/mayflower-blocks#88]
The contrast between Link text color and the background of a Collapsible with "Info" color selected is far too low and fails WCAG a11y standards.

This poor contrast is consistent across web browsers and mobile devices.

Site that demonstrates behavior: [https://staging-bc2022redesigncontentstage-stage.kinsta.cloud/english/a-everything-mayflower-block-page/](https://staging-bc2022redesigncontentstage-stage.kinsta.cloud/english/a-everything-mayflower-block-page/)

![Image](https://github.com/user-attachments/assets/002c2050-66af-42a0-a619-3eaa11766958)
![Image](https://github.com/user-attachments/assets/5d8420f8-1b11-49de-92fa-2d523da5a4b6)
